apps/melding-form/src/app/(general)/samenvatting/_components/AttachmentImage.module.css

```diff
@@ -0,0 +1,10 @@
+.image {
+  margin-block-end: var(--ams-space-m);
+  max-inline-size: 24rem;
+}
+
+.loading {
+  composes: image;
+  aspect-ratio: var(--ams-aspect-ratio-16-9);
+  background-color: var(--ams-color-separator);
```

Comment:
"color-separator" is not a very semantic name here right? Can't you just set the grey color?

Reply:
No, it isn't very semantic indeed. The Design System doesn't expose this light grey color in any other way though. We could hardcode the value, but that would mean we wouldn't receive any changes to the color anymore.

I think this is the best compromise for now, and I'll add this to the ADS backlog.